src/js/views/programs/program/flow/program-flow.scss

```diff
@@ -1,8 +1,6 @@
 .program-flow__frame {
   display: flex;
-  height: 100%;
-  overflow-x: hidden;
-  overflow-y: auto;
+  flex: 1;
 }
```

Comment:
This page doesn't use the patient sidebar. But I updated this to remain consistent with what's used for the other pages (i.e. `.patient__frame`, `.patient-flow__frame`, etc.).

Same goes for `src/js/views/programs/program/program.scss`.